src/Stratis.Bitcoin.Features.BlockStore/AddressIndexing/AddressIndexerOutpointsRepository.cs

```diff
@@ -110,7 +110,7 @@ public void RecordRewindData(AddressIndexerRewindData rewindData)
             }
         }
 
-        /// <summary>Deletes rewind data originated at height lower than <paramref name="height"/>.</summary>
+        /// <summary>Deletes rewind data items that were originated at height lower than <paramref name="height"/>.</summary>
```

Comment:
I meant the comment in line 119 actually

Reply:
fixed